xcode/Subconscious/Shared/Components/AppUpgradeView.swift

```diff
@@ -0,0 +1,175 @@
+//
+//  AppUpgradeView.swift
+//  Subconscious
+//
+//  Created by Gordon Brander on 3/22/23.
+//
+
+import SwiftUI
+import Combine
+import ObservableStore
+
+/// Displays information to the user when app migration / rebuild happening.
+struct AppUpgradeView: View {
+    private let bgGradient = LinearGradient(
+        gradient: Gradient(
+            colors: [.brandBgTan, .brandBgGrey, .brandBgBlush]
+        ),
+        startPoint: .topLeading,
+        endPoint: .bottomTrailing
+    )
```

Comment:
This view doesn't handle light vs dark mode currently, in fact it barely changes at all between the two modes.

We could force this screen to be theme agnostic, which requires fixing a few label colours etc. as seen here:

Light:

<img width="497" alt="image" src="https://user-images.githubusercontent.com/5009316/229393833-8fd3ffe9-4950-4fed-a21e-e9d734723310.png">

Dark:

<img width="487" alt="image" src="https://user-images.githubusercontent.com/5009316/229393810-de669a22-81f8-4512-9054-a4a6ea5c91da.png">

(main issue is the Upgrading... label + button visibility). 

We could also define two gradients the way we do for the FAB and make this view respond fairly easily.